docs/guides/liquidations.md

```diff
@@ -0,0 +1,84 @@
+# Liquidation Guide
+
+This guide instructs users how to process a liquidation of an under-collateralized loan using polkadot.js.
+
+At the end of this guide you will know how to:
+
+- [x] [Funded your wallet with the debt token and gas token](#fund-wallet)
+- [x] [Liquidated a borrow position using polkadot.js](#liquidate-borrow-position)
+- [x] [Withdraw the received collateral tokens](#withdraw-collateral-optional)
+- [x] [Batch liquidations](#batch-liquidations)
+
+## Steps
+
+### **Fund wallet**
+
+1. The wallet needs as small amount of KINT tokens to pay for the gas fees.
+2. The wallet needs to be funded with the amount of `debt tokens` which shall be repaid. E.g. if the liquidator aims to liquidate a debt position of 1,000 USDT, the wallet needs to have at least this amount in the respective debt currency, in that case, USDT.
+
+All tokens that can be used in the money market can also be traded on Kintsugi DEX. For alternative options see [Other DEXs](#other-dexs)
+
+### **Liquidate borrow position**
+
+1. Go to [https://polkadot.js.org/apps/#/extrinsics](https://polkadot.js.org/apps/#/extrinsics) and select Kitnsugi network
+2. Set up the parameters for the call
+    1. **using the selected account**: `your_accound_address` (Kintsugi address, starting with ‘a3…’)
+    2. **submit the following extrinsic**: `loans.liquidateBorrow()`
+    3. **borrower:** `target_account_address` (Kintsugi address, starting with ‘a3…’)
+    4. **liquidationAssetId:** Currency which needs to be repaid
+        1. Select `Token` for KINT, kBTC, KSM
+        2. Select `ForeignAsset` for other tokens
+            1. see [here](#asset-registry) for `id:token` mapping
+    5. **repayAmount:** `amount_to_be_repaid` , note that the amount depends on the number of decimals the token uses. [See](#asset-registry) here for details.
+    6. **collateralAssetId:** The underlying of the currency to be received. E.g: For kBTC the liquidator will receive the lend token of kBTC which can be redeemed for kBTC (see [withdraw collateral](#withdraw-collateral-optional))
+        1. Select `Token` for KINT, kBTC, KSM
+        2. Select `ForeignAsset` for other tokens
+            1. see [here](#asset-registry) for `id:token` mapping
+    
+3. Submit the transaction
+
+### **Example**
+
+This example would liquidate a $1,000 USDT position in order to receive kBTC as collateral.
+
+![Liquidate Borrow Extrinsic](../_assets/img/guide/liquidate-borrow-extrinsic.png)
+
+### **Withdraw collateral (optional)**
```

Comment:
They can use the UI for this step, since any qToken balance will show up under "My Lend Positions" in the lending page (including the qTokens received from the liquidation). You can just link this: https://docs.interlay.io/#/guides/lending?id=_7-withdraw-a-deposit

Reply:
But I see that the next section makes reference to the withdrawal extrinsic so you can also just link to the UI guide and not remove this section